spec/namespaces/security.yaml

```diff
@@ -1245,13 +1314,19 @@ components:
     security.cache@501:
       content:
         application/json:
-          schema:
-            $ref: '../schemas/security._common.yaml#/components/schemas/MethodNotImplemented'
+          schema: null
+          # $ref: '../schemas/security._common.yaml#/components/schemas/MethodNotImplemented'
     security.change_password@200:
       content:
         application/json:
           schema:
             $ref: '../schemas/security._common.yaml#/components/schemas/Ok'
+    security.change_password@403:
+      description: ''
+      content:
+        application/json:
+          schema: null
+          # $ref: '../schemas/security._common.yaml#/components/schemas/Forbidden'
```

Comment:
This doesn't look intended?

Reply:
I have kept this as comment to make it easier to replace once support for status other than 200s is added for schema validation. Let me know if I should still remove it. 

Providing schema: null allows to check for 403s without providing any response payload schema, and in turn unblocks this PR since some APIs are admin only.

Reply:
I see, good idea. Let's use a `TODO`, e.g. add `TODO: add #ref ....` or something like that?